code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -491,3 +492,22 @@
 	qdel(theme)
 	reactivearmor_cooldown = world.time + reactivearmor_cooldown_duration
 	return FALSE
+
+/obj/item/clothing/suit/armor/reactive/ectoplasm
```

Comment:
might want to add a bounty for the refined core to make it in-line with other cores